SourceryRuntime/Sources/TypeName.swift

```diff
@@ -66,23 +66,24 @@ public final class TypeName: NSObject, AutoCoding, AutoEquatable, AutoDiffable,
             self.isOptional = false
         } else {
             name = name.bracketsBalancing()
+            let trimmedInoutName = name.trimmingPrefix("inout ")
```

Comment:
I think this should change `name` value, it's later checked for prefix, so `inout` should be trimmed.